### ISSUE TITLE
We need docs.

### ISSUE THREAD

**Issue by forki** [fsprojects/Chessie#3]
I started by adding [XML docs](http://fsprojects.github.io/Chessie/reference/chessie-rop.html). Would like to see more docs and links to videos and articles.
Also samples (maybe taken from Paket).

**Comment by pblasucci** [comment]
FWIW, I'm getting a lot of errors an weird output trying to create API docs (on OS X). I realize it's probably an issue with FSharpFormatting and `inline` functions. But I'm raising it here for visibility (and in case anyone already has a workaround).

**Comment by CumpsD** [comment]
Would it be possible to add an example on the async bits to the docs? Trying to wrap my head around how to use them properly, but can't figure it out

**Comment by pblasucci** [comment]
I agree. This would be great. I'm not certain I've ever properly understood the use cases for Chessie's async bits. I'm pretty sure Kai Frederich added that originally... Maybe he can write up a quick example or two?

Paul Blasucci
